var/spack/repos/builtin/packages/p4est/package.py

```diff
@@ -42,8 +42,8 @@ class P4est(Package):
     depends_on('mpi')
     depends_on('zlib')
 
-    def install(self, spec, prefix):
-        options = [
+    def configure_args(self):
```

Comment:
Oops, good catch. I realize this conflicts with your #2839. Do you want me to remove this file from my changes, or do you want to close #2839? Or do you want to play chicken and see whose PR gets merged first? 😆